scripts/ci/pr-bot/test/reviewerConfigTest.ts

```diff
@@ -0,0 +1,218 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var assert = require("assert");
+var fs = require("fs");
+const { ReviewerConfig } = require("../shared/reviewerConfig");
+const configPath = "test-config.yml";
+const configContents = `labels:
+- name: "Go"
+  reviewers: ["testReviewer1", "testReviewer2"]
+  exclusionList: ["testReviewer3"] # These users will never be suggested as reviewers
+# I don't know the other areas well enough to assess who the normal committers/contributors who might want to be reviewers are
+- name: "Java"
+  reviewers: ["testReviewer3", "testReviewer2"]
+  exclusionList: [] # These users will never be suggested as reviewers
+- name: "Python"
+  reviewers: ["testReviewer4"]
+  exclusionList: [] # These users will never be suggested as reviewers
+fallbackReviewers: ["testReviewer5", "testReviewer1", "testReviewer3"] # List of committers to use when no label matches
+`;
+describe("ReviewerConfig", function () {
+  before(function () {
+    if (fs.existsSync(configPath)) {
+      fs.rmSync(configPath);
+    }
+    fs.writeFileSync(configPath, configContents);
+  });
+
+  after(function () {
+    fs.rmSync(configPath);
+  });
+
+  describe("getReviewersForLabels()", function () {
+    it("should return all reviewers configured for all labels", function () {
+      const config = new ReviewerConfig(configPath);
+      const reviewersForLabels = config.getReviewersForLabels(
+        [{ name: "Go" }, { name: "Java" }],
+        []
+      );
+      assert(
+        reviewersForLabels["Go"].indexOf("testReviewer1") > -1,
```

Comment:
Using `find` instead of `indexOf` would be cleaner because it returns null which is false.